package.json

```diff
@@ -89,6 +86,7 @@
   },
   "dependencies": {
     "@faker-js/faker": "^8.4.1",
+    "@aws-sdk/client-sqs": "^3.592.0",
```

Comment:
This and the corresponding dev dependency `aws-sdk-client-mock` are not used from what I can see, did these slip in from the template project?